### PR TITLE
14.0 mig product custom info

### PR DESCRIPTION
**Pull request by ilyasProgrammer**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
LGTM

**Review by elvise — APPROVED** [review]
LGTM

**Comment by francesco-ooops** [comment]
@OCA/product-maintainers merge?

File: product_custom_info/README.rst [hunk 1]
```diff
@@ -1,77 +0,0 @@
...
-===================
-Product Custom Info
-===================
-
-This module extends products to allow having custom properties on them.
-
-Installation
-============
-
-* You depend from module ``base_custom_info``, found in `OCA/server-tools
```
Comment (rousseldenis) [inline product_custom_info/README.rst]: @ilyasProgrammer If you remove the README you need to fill in all the corresponding readme fragments (INSTALL/CONFIGURE/CONTRIBUTORS...).

**Review by rousseldenis — CHANGES_REQUESTED** [review]
@ilyasProgrammer Thanks for this. Please add missing readme fragments

**Comment by francesco-ooops** [comment]
@rousseldenis good now? :)

**Comment by rousseldenis** [comment]
> @rousseldenis good now? :)

Nope.

Please move each README section to corresponding section in readme/*.rst

See https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by rousseldenis** [comment]
@ilyasProgrammer Ok, better now. Could you squash additional commits in the migration one ?

**Comment by francesco-ooops** [comment]
@rousseldenis code ok for merge?

**Comment by francesco-ooops** [comment]
@OCA/product-maintainers can this one be reviewed/merged? thanks!

**Comment by francesco-ooops** [comment]
@rousseldenis @JordiBForgeFlow sorry to tag you directly, can you take a look when you have some time?

**Comment by francesco-ooops** [comment]
@OCA/product-maintainers :)
image

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**